Create metadata table

adilahiri opened the pull request:
<!--Hi there, thanks for your contribution! Please take a moment to fill out this template to facilitate the review of your pull request.-->

### Purpose/implementation Section

#### What scientific question is your analysis addressing?

Create Table S1 for future manuscript to contain the histologies file along with a README




#### What was your approach?
Adapted code from:
https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/tables/output_tables.Rmd


#### What GitHub issue does your pull request address?
[Issue#459
](https://github.com/PediatricOpenTargets/ticket-tracker/issues/459)

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.

#### Which areas should receive a particularly close look?
Please review [TableS1-histologies.xlsx](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/blob/create-metadata-table/tables/results/TableS1-histologies.xlsx) and the script [output_tables.nb.html](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/blob/create-metadata-table/tables/output_tables.nb.html)


#### Is there anything that you want to discuss further?
NA

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes


### Results

#### What types of results are included (e.g., table, figure)?
Table


#### What is your summary of the results?



#### Reproducibility Checklist

<!-- Check all those that apply or remove this section if it is not applicable.-->

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

<!-- Please review https://github.com/AlexsLemonade/OpenPBTA-analysis#documenting-your-analysis -->

- [ X] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [ X] The analytical code is documented and contains comments.

<!-- IF YOUR PULL REQUEST IS A DATA RELEASE, PLEASE REMOVE THE [HTML COMMENT TAG](https://html.com/tags/comment-tag/) FROM THE SECTION BELOW AND COMPLETE THE CHECKLIST-->

<!-- 
### Data Release Checklist
- [ ] Is the table in doc/data-file-descriptions.md up to date?
- [ ] Is doc/data-format.md up to date?
- [ ] Is doc/release-notes.md up to date?
- [ ] Is download-data.sh up to date?
- [ ] Was download-data.sh tested and did it complete without error?
-->

jharenza reviewed (changes requested):
Hi @adilahiri thanks for starting this. Can you make sure all columns from v11 are represented in the readme? Eg - the methylation information. Thanks!

adilahiri commented:
@jharenza thank you for pointing out the missing methylation columns. I have added them into the README now. 

Following are the extra columns that have been added to README. I have added the description based on my understand, but they can be improved especially for the classification scores by specifying the type of score (accuracy, F-1, roc_auc_score etc). 

| Histology column| Definition| 
| :---         |     :---:      |      
| cns_methylation_class  | methylation-based CNS tumour classes  | 
| cns_methylation_class_score   | class score | 
| cns_methylation_subclass   | methylation-based CNS tumour sub-classes      | 
| cns_methylation_subclass_score   | subclass score     | 
| gtex_group   | Tissue Type| 
| gtex_subgroup   | Tissue Subtype | 
| RF_purity_ABSOLUTE   | purity score based on ABSOLUTE dataset | 
| RF_purity_ESTIMATE   | purity score based on ESTIMATE dataset | 
| RF_purity_LUMP   |purity score based on LUMP dataset |

jharenza commented on tables/output_tables.Rmd:
```diff
@@ -0,0 +1,118 @@
+---
+title: "Tables output for manuscript"
+author: "Aditya Lahiri,Eric Wafula and Jo Lynne Rokita"
```
```suggestion
author: "Aditya Lahiri"
```

jharenza commented on tables/output_tables.Rmd:
```diff
@@ -0,0 +1,118 @@
+"primary_site","Bodily site(s) from which specimen was derived","text",
+"race","Patient reported race",paste(unique(histology_df$race), collapse = "; "),
+"reported_gender","Patient reported gender",paste(unique(histology_df$reported_gender), collapse = "; "),
+"RF_purity_ABSOLUTE","purity score based on ABSOLUTE dataset","numeric",
+"RF_purity_ESTIMATE","purity score based on ESTIMATE dataset","numeric",
+"RF_purity_LUMP","purity score based on LUMP dataset","numeric",
```
```suggestion
"RF_purity_ABSOLUTE","tumor purity fraction estimated using ABSOLUTE","numeric",
"RF_purity_ESTIMATE","tumor purity fraction estimated using ESTIMATE dataset","numeric",
"RF_purity_LUMP","tumor purity fraction estimated using LUMP","numeric",
```

jharenza commented on tables/output_tables.Rmd:
```diff
@@ -0,0 +1,118 @@
+"broad_histology","Broad WHO 2016 classification of cancer type",paste(unique(histology_df$broad_histology), collapse = "; "),
+"cancer_group","Harmonized cancer groupings for plots",paste(unique(histology_df$cancer_group), collapse = "; "),
+"cancer_predispositions","Reported cancer predisposition syndromes",paste(unique(histology_df$cancer_predispositions), collapse = "; "),
+"cns_methylation_class","methylation-based CNS tumour classes",paste(unique(histology_df$cns_methylation_class), collapse = "; "),
+"cns_methylation_class_score","class score","numeric",
+"cns_methylation_subclass","methylation-based CNS tumour sub-classes",paste(unique(histology_df$cns_methylation_subclass), collapse = "; "),
+"cns_methylation_subclass_score","subclass score","numeric",
```
```suggestion
"cns_methylation_class","methylation-based CNS tumor class",paste(unique(histology_df$cns_methylation_class), collapse = "; "),
"cns_methylation_class_score","methylation-based CNS tumor class score","numeric",
"cns_methylation_subclass","methylation-based CNS tumor sub-class",paste(unique(histology_df$cns_methylation_subclass), collapse = "; "),
"cns_methylation_subclass_score","methylation-based tumor subclass score","numeric",
```

jharenza commented on tables/run-manuscript-tables.sh:
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Module authors: Aditya Lahiri,Eric Wafula and Jo Lynne Rokita
```
```suggestion
# Module author: Aditya Lahiri
```

jharenza reviewed (approved):
few comments, I will commit, rerun, and approve this so we can hand off to NCI

jharenza commented:
I will merge this without checks since it doesn't run in GA
